docs/quick.rst

```diff
@@ -1,20 +1,79 @@
 Quick Start
 ===========
 
-The most common usage of the astromon data products is the list of cross-matches::
+Pre-computed queries
+--------------------
+
+A common usage of the astromon data products is to get the *standard* pre-computed cross-matches
+using :any:`db.get_cross_matches`. This function takes a `name` argument that specifies the
```

Comment:
Only one of those examples failed for me, so it would be nice to know what error you got.

I fixed the one that was failing. It was because of inconsistency between the output from db.get_cross_matches and `cross_match.cross_match`.

Right when you were leaving your comments I added a new function. All the functionality you want should already be there.